src/main/java/org/akaza/openclinica/controller/dto/SdvItemDTO.java

```diff
@@ -19,6 +19,15 @@
     private String lastModifiedUserLastName;
     private String sdvStatus;
     private int openQueriesCount;
+    private boolean isActionsAllowed;
```

Comment:
Instead of calling it actionsAllowed, I would just call it "isCalculateItem" and then have the UI handle it properly. I don't think we want to mask the reason for why the actions allowed from the UI.

Reply:
Sure Shiva, I will change that